packages/stark-ui/src/modules/slider/components/_slider-theme.scss

```diff
@@ -61,14 +63,14 @@
       border-radius: 50%;
     }
   }
-  &.horizontal {
+  & .horizontal {
```

Comment:
I do not see the slider in the showcase but are you sure this change won't break the layout?

As we discussed `&` is different in SCSS than in PCSS

Reply:
Actually, I adapted this because previously, it was like this:
```html
<div class="stark-slider" [ngClass]="{'horizontal': isHorizontal, 'vertical': !isHorizontal}"></div>
```

And now it is like this:
```html
<stark-slider class="stark-slider">
  <div [ngClass]="{'horizontal': isHorizontal, 'vertical': !isHorizontal}"></div>
</stark-slider>
```

That's why I adapted the CSS from `.stark-slider.horizontal` to `.stark-slider .horizontal`

Reply:
ok but then the `&` can be removed :-)